app/profile/user.py

```diff
@@ -0,0 +1,20 @@
+from app.infra.models.user import User
```

Comment:
I would delete this file, and move `app/auth/repos.py` to the this profile dir since that has the same code effectively

Reply:
Should I create `app/profile/user.py` or just directly import `from app.auth.repos import UserRepo`

Reply:
it should be `app/profile/repos.py`. And the places in the `auth` package would import from `app.profile.repos`